From 8eda8f2f266c6568dc982515a0eff67f12e49248 Mon Sep 17 00:00:00 2001
From: stoffu <stoffu@protonmail.ch>
Date: Fri, 19 Oct 2018 18:21:26 +0900
Subject: [PATCH] enable static libraries on mingw

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index d702461e..abfd88b0 100644
--- a/configure.ac
+++ b/configure.ac
@@ -261,7 +261,7 @@ case "${host_os}" in
         libzmq_dso_visibility="no"
 
         if test "x$enable_static" = "xyes"; then
-            AC_MSG_ERROR([Building static libraries is not supported under MinGW32])
+            CPPFLAGS="-DZMQ_STATIC $CPPFLAGS"
         fi
         ;;
     *cygwin*)
-- 
2.14.1

